draft/0000-new-branching-rules-2023.md

```diff
@@ -0,0 +1,41 @@
+# 2023 update to naming and rules regarding branches in OpenMage
+
+## Summary
+
+Rename the active branches and change which one if the default.
+
+## Motivation
+
+### 1.
+Up to now differences between v19 and v20 were minor and the default branch was 1.9.4.x. Almost everything was committed to the 1.9.4.x and, at release time, forward ported to the 20.0 branch.
+This was ok for the first years of the project but I think it should be changed now in 2022.
```

Comment:
Change 2022 -> 2023. Hopefully this is implemented and we will not still be discussing in 2024. :)

Reply:
typo fixed